adr/cli.py

```diff
@@ -108,6 +106,7 @@ def run_recipe(recipe, args, fmt='table'):
 
 
 def cli(args=sys.argv[1:]):
+    from adr.query import set_active_data_url
```

Comment:
nit: Please move this to the top (under the `adr.formatter` import)

Reply:
Thanks. I deliberately put the import inside the function, otherwise, a circular import will be raised. What shall I do?

Reply:
Oh, good catch! In that case I think your solution of just putting it in the function is good for now. I'll try and refactor things a bit such that the circular import doesn't happen in the future, but it isn't really related to this issue.